.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
```

Comment:
Is this intended to be in this PR. I don't have a problem with it, just want to point it out.

Reply:
I opened #702 to fix these issues, so some of these changes will disappear when I merge or rebase master.